keras2onnx/common/onnx_ops.py

```diff
@@ -69,21 +65,17 @@ def _apply_pointwise_operation(scope, op_type, input_names, output_name, contain
 
     container.add_node(op_type, input_names, output_name, op_version=op_version, name=name, **attrs)
 
-
 def apply_abs(scope, input_name, output_name, container, operator_name=None):
     _apply_unary_operation(scope, 'Abs', input_name, output_name, container, operator_name=operator_name)
 
-
 def apply_add(scope, input_names, output_name, container, operator_name=None, axis=None, broadcast=None):
     _apply_basic_numerical_operation(scope, 'Add', input_names, output_name, container, operator_name=operator_name,
                                      axis=axis, broadcast=broadcast)
 
-
 def apply_batch_norm(scope, input_names, output_names, container, operator_name=None,
                      epsilon=None, is_test=None, momentum=None, spatial=None):
     name = _create_name_or_use_existing_one(scope, 'BatchNormalization', operator_name)
-
-    attrs = {'name': name, 'epsilon': epsilon, 'momentum': momentum, 'spatial': spatial}
+    attrs = {'name': name, 'epsilon': epsilon, 'momentum': momentum}
 
     if container.target_opset < 6:
```

Comment:
Can we write like 
`if container.target_opset < 9:      attrs['spatial'] = spatial`
This way we only need to assign attrs['spatial'] once, compared you need write it 3 times here. Same for is_test

Reply:
Thanks for the feedback, @jiafatom! Resolved in latest update.